### ISSUE TITLE
Be able to connect using a self signed certificate

### ISSUE THREAD

**Issue by aholzel** [eclecticiq/intelligence-center-app-splunk-ta#19]
The app now only lets you connect if you have a valid certificate installed on your TIP. Please make a options to also allow self signed or other wise "invalid" certificates. This used to be an checkbox in the old app. This is especially useful in a test setup.

**Comment by aholzel** [comment]
The workaround for the initial connection problem when trying to add a account is to change line 405 in `<SPLUNK HOME>/etc/apps/TA-eclecticiq/bin/validator/validate.py` from 
`verify=True,` to `verify=False,`

This however doesn't solve the later connection problems when the Splunk app is trying to connect to the TIP to get the feed data.
```
2022-09-16 13:19:46,958 ERROR pid=13750 tid=MainThread file=base_modinput.py:log_error:309 | Error: 
HTTPSConnectionPool(host='tip', port=443): Max retries exceeded with url: /api/v1/entities?
limit=200&filter%5B%3Elast_updated_at%5D=2022-09-01T00%3A00%3A00.1&filter%5Boutgoing_feeds%5D=6&sort=-
last_updated_at (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify 
failed: self signed certificate (_ssl.c:1106)')))
```

**Comment by himynamesdave** [comment]
Hey @aholzel ,

So I'm planning to add support for this in the next release.

Seeing as it's hard to validate certs from a closed authority, my proposed solution is similar to yours;

Allow user to set a flag, ignore certificate warning, in the setup. If true, any cert warnings will be ignored.

Is this acceptable for you?

**Comment by aholzel** [comment]
Yes that whould be a great solution. That way a user can choose if he/she want to allow self signed certificates that might be used in a test/lab enviroment.

**Comment by himynamesdave** [comment]
fixed in 4bc369b
